client/curriculum/intro/despre-noi.tsx

```diff
@@ -3,12 +3,16 @@ import { useRouter } from 'next/router';
 import SEOTags from '~/components/SEOTags';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.